client/src/components/campaignCreatorComponents/CampaignTypeConfig.tsx

```diff
@@ -92,3 +92,18 @@ import rawResourceListData from '../../static_data/resource_list.json';
 export type TypeResourceListData = { [K in keyof typeof rawResourceListData]: string[] };
 
 export const resourceListData: TypeResourceListData = rawResourceListData;
+
+//------------------------------------------------------------
+// --- Locations Json ---
+import rawLocationsData from '../../static_data/locations.json';
```

Comment:
Tempting as it might seem I wouldn't recommend importing in the middle of a file.  Do it all at the top

Reply:
makes sense